fix: open legal links in new tab

mrcattington opened the pull request:
# Description

Added `legalLinksNewTab` option to allow Terms and Privacy Policy links to optionally open in new tabs.

Changes:
- Added new optional configuration option `legalLinksNewTab`
- When enabled, legal links open in new tabs with proper security attributes
- Maintains backward compatibility by defaulting to current behavior

This improves UX by giving developers the choice of whether legal links should open in the same tab or a new tab.
## Type of change

- [ ] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Associated Issues

For Linear issues: Closes APKT-xxx
For GH issues: closes #...

# Showcase (Optional)

If there is a UI change include the screenshots with before and after state.
If new feature is being introduced, include the link to demo recording.

# Checklist

- [ ] Code in this PR is covered by automated tests (Unit tests, E2E tests)
- [x] My changes generate no new warnings
- [x] I have reviewed my own code
- [x] I have filled out all required sections
- [ ] I have tested my changes on the preview link
- [ ] Approver of this PR confirms that the changes are tested on the preview link

arein commented on apps/laboratory/.env.example:
```diff
@@ -1,5 +1,5 @@
 # Obtain a project ID from https://cloud.walletconnect.com
-NEXT_PUBLIC_PROJECT_ID=""
+NEXT_PUBLIC_PROJECT_ID="ee7410d12dfbe505287359a4dd81ac22"
```
```suggestion
NEXT_PUBLIC_PROJECT_ID=""
```

mrcattington replied:
Yes, I did expect it to be default behavior. I'll update PR to make open in tabs by default.

arein reviewed (commented):
Currently it opens in the same window? that's a bug to me - should just change the default behavior?

mrcattington commented:
I've updated the code to make opening links in new tabs the default behavior instead of having it as an optional configuration, and removed all the optional flag code. I've also reverted the project ID in the .env.example file.

mrcattington commented:
Not trying to be annoying, just haven’t seen any movement on this in a while. I was hoping it would be included in the next release. Thank you!
